### ISSUE TITLE
sequence is not function

### ISSUE THREAD

**Issue by loun4** [jameslnewell/xhr-mock#91]
According to the documentation 

```js
import mock, {sequence} from 'xhr-mock';

mock.setup();

mock.post('/', sequence([
  {status: 200}, // the first request will receive a response with status 200
  {status: 500}  // the second request will receive a response with status 500
                 // if a third request is made, no response will be sent
  ]
));
```
The following error is thrown `TypeError: (0 , _xhrMock.sequence) is not a function`
Is there another way to use sequential calls?

**Comment by loun4** [comment]
As `sequence` is not released yet (#90), i resolved my issue using `once`

```javascript
mock.post('/', once(
  {status: 200}, // the first request will receive a response with status 200
));

mock.post('/', once(
  {status: 500}, // the second request will receive a response with status 500
));
```

**Comment by jameslnewell** [comment]
Apologies! Try `xhr-mock@2.5.0`.
